### DISCUSSION TITLE
AVI File not the right length after cutting

### DISCUSSION THREAD

**Discussion by kevin00xxl** [mifi/lossless-cut#820] [Q&A]
After i cut an .AVI file with the original duration of 1 hour and 11 minutes, when i cut it to 55 minutes (i tried keyframe cut and normal cut) the file after exporting still says 1 and 11 minutes. When i play the video after te point i had cut it it just freezes the last frame.

I however tried the "Fix incorrect duration" option in LosslessCut, after which i reimported the same file, did the same cut and the length now is perfect.

My question is; is there any loss in quality if you first do the "Fix incorrect duration" and then reimport, cut and export the file?

![fix incorrect duration](https://user-images.githubusercontent.com/62187554/130415572-16ee1693-29a3-4cb9-bb9f-4584ebadeeff.PNG)

**Comment by mifi** [comment]
That option will do a lossless copy, so there will be no quality loss. You may lose some metadata or extra tracks though.

Code is here:
https://github.com/mifi/lossless-cut/blob/75261990cd20b8d52aefa72dd6129fe752f0fa15/src/hooks/useFfmpegOperations.js#L375
